ocaml/src/tools/etcd.ml

```diff
@@ -0,0 +1,48 @@
+(*
```

Comment:
This file should be moved to arakoon and exposed in it's client lib?
And a similar remark for Prelude.Url ?